packages/beacon-dapp/src/dapp-client/DAppClient.ts

```diff
@@ -435,9 +435,11 @@ export class DAppClient extends Client {
       }
     })()
 
-    await (isWaiting
-      ? this.events.emit(BeaconEvent.WC_ACKNOWLEDGE_PENDING, { walletInfo })
-      : this.events.emit(BeaconEvent.WC_ACKNOWLEDGE_RECEIVED, { walletInfo }))
+    console.log('walletInfo: ', walletInfo)
+
+    setTimeout(async () => await this.events.emit(BeaconEvent.HIDE_UI, ['alert']))
```

Comment:
I think we don't need to "async/await" here.

Why do we need to hide the UI? Does updating the toast not automatically dismiss the old toast?